kuma/users/jinja2/users/ban_user_and_cleanup.html

```diff
@@ -0,0 +1,67 @@
+{% extends "base.html" %}
+
+{% block body_attributes %}{% endblock %}
+
+{% block bodyclass %}user ban-user{% endblock %}
+
+{% block title %}{{ _('Ban') }} {{ detail_user }}{% endblock %}
+{% set styles = ('dashboards', 'jquery-ui') %}
+{% set classes = 'compare' %}
+
+{% block site_css %}
+  {{ super() }}
+  {% stylesheet 'users' %}
+  {% stylesheet 'user-banned' %}
+{% endblock %}
+
+{% block content %}
+<section id="content">
+{# TODO: Add an if block if the user has already been identified as a spammer #}
+
+{% if not user_banned %}
+  <h1>{{ _('Identify') }} <a href="{{ detail_user.get_absolute_url() }}">{{ detail_user }}</a> {{ _('as a spammer?') }}</h1>
+{% else %}
+  <h1><a href="{{ detail_user.get_absolute_url() }}">{{ detail_user }}</a> has already been banned. You may review their revisions below.</h1>
```

Comment:
This is a long string for a ``<h1>`` element:

<img width="891" alt="has_been_banned" src="https://cloud.githubusercontent.com/assets/286017/16231651/c43bfcde-378c-11e6-94ea-670a070ce232.png">

Maybe something shorter like "(user) has been banned", with a follow up paragraph?  This can wait until a later phase, when @stephaniehobson can give feedback.